scripts/bootstrap.ps1

```diff
@@ -37,12 +39,30 @@ if ((get-module psake) -eq $null) {
 }
 
 if ((get-module psake) -eq $null) {
-    # Still not loaded, let's try the various NuGet caches
-    $locals = .\nuget locals all -list
+    # Not yet loaded, try to load it from the chocolatey installation library
+    TryLoad-Psake $env:ProgramData\chocolatey\lib
+}
+
+if ((get-module psake) -eq $null) {
+    # Still not loaded, let's download it and then look in the various NuGet caches
+
+    pushd .\src\sestest
+    dotnet add package psake | Out-Null
+    popd
+
+    $locals = $null
+    $nuget = get-command nuget -ErrorAction SilentlyContinue
+    $dotnet = get-command dotnet -ErrorAction SilentlyContinue
+    if ($nuget -ne $null) {
+        $locals = & $nuget locals all -list
+    } elseif ($dotnet -ne $null) {
+        $locals = & $dotnet nuget locals all --list
+    }
+
     foreach($local in $locals)
     {
-        $index = $local.IndexOf(":")
-        $folder = $local.Substring($index + 2)
+        $index = $local.LastIndexOf(":")
+        $folder = $local.Substring($index - 1)
```

Comment:
Is this exploiting the assumption that the path will be of the form <driveletter>:<path>? Does it need to work on Linux?

Reply:
Yes, this does need to work on Linux. I'll have to tweak this to work cross-platform.